kloppy/_providers/statsperform.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from kloppy.domain import TrackingDataset
+from kloppy.infra.serializers.tracking.statsperform import (
+    StatsperformDeserializer,
+    StatsperformInputs,
+)
+from kloppy.io import FileLike, open_as_file
+
+
+def load(
+    meta_data: FileLike,  # Statsperform MA1 file - XML - single game, live data & lineups
+    raw_data: FileLike,  # Statsperform MA25 file
```

Comment:
I think we should try to make more explicit that the `raw_data` file is a text file rather than an XML file. The MA25 API endpoint returns a response in either XML or JSON format, which includes a download link to the text files that contain the actual tracking data.